lib/Parse/ParseExpr.cpp

```diff
@@ -3226,7 +3234,7 @@ ParserStatus Parser::parseExprList(tok leftTok, tok rightTok,
     // First check to see if we have the start of a regex literal `/.../`. We
     // need to do this before handling unapplied operator references, as e.g
     // `(/, /)` might be a regex literal.
-    tryLexRegexLiteral(/*mustBeRegex*/ false);
+    tryLexRegexLiteral(/*forUnappliedOperator*/ true);
```

Comment:
`forUnappliedOperator` might be confusing. More like `canBeUnappliedOperator` because we don't even know there's a `/` in this token at this point.

Reply:
Actually changed it so we don't call `tryLexRegexLiteral` unless we know we have an unapplied operator. This lets us avoid re-lexing twice for prefix operators, as they only need to be re-lexed in `parseExprUnary`. How does it look?